src/app.ts

```diff
@@ -35,7 +35,16 @@ app.get(
 app.use('/api', apiApp);
 
 async function initDatabase(): Promise<void> {
-  if (!getConnection() && process.env.NODE_ENV !== 'test') {
+  let hasConnection;
+
+  try {
+    getConnection();
+    hasConnection = true;
+  } catch (err) {
+    hasConnection = false;
+  }
+
+  if (!hasConnection && process.env.NODE_ENV !== 'test') {
```

Comment:
Instead of using this bool, could we just return from the success block of the try/catch?

Reply:
Yeah, that's better lol